server/src/test/java/org/apache/kylin/rest/util/ValidateUtilTest.java

```diff
@@ -48,6 +48,9 @@
 
     @Test
     public void testCheckIdentifiersExists() throws IOException {
+        String project = "/asd/../_asd33b";
+        project = ValidateUtil.convertStringToBeAlphanumericUnderscore(project);
```

Comment:
Could we add some assertEqual for this `project`?

Reply:
@hit-lacus This is some code I submit carelessly, I have removed it